examples/video_channel_create.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+import sys
+
+import jwplatform
+
+
+def create_channel(api_key, api_secret, channel_type='manual', **kwargs):
```

Comment:
Hey Sergey.  we just wanted simple examples.  we followed the existing pattern you already set in the original example to just accept an api_key and api_secret in the method.

the end user can decide to decouple the code to a common client if they wish.  we didn't want to complicate things for the end user.